### ISSUE TITLE
Genre classifications

### ISSUE THREAD

**Issue by stevenstetzler** [The-Wayfinder-Society/wayfinder-app#8]
Found that some electronic genres with "Hardcore" in the name are labeled metal:
- Australian Hardcore
- Digital Hardcore
- Happy Hardcore

**Comment by stevenstetzler** [comment]
Although one of the artists in this classification, "Machine Girl", is labeled [Digital Hardcore](https://en.wikipedia.org/wiki/Digital_hardcore), which really is a fusion between electronic and metal influences. Maybe worth thinking about how to deal with blended genres? Or might be too much hassle to handle all these edge cases...

**Comment by hvbish** [comment]
Yes, I agree that we should refine the way genres are handled. In the original visualization I reclassified a good number of specific cases where I felt the umbrella genre had been assigned incorrectly, but I think that to really deal with genres properly we'll need to do more than handle a few edge cases. The task of genre classification is quite complex and subjective and messy; virtually all genres are a blend of some sort. Pop and Rock especially have tons of overlap. So we may eventually want to completely rethink the way the umbrella genres themselves are defined.

To take your example, Digital Hardcore has elements of Electronic, punk (which falls under Rock), and Metal (which is really a sub-genre of Rock too, even though we chose to keep it separate). It doesn't fit neatly into any of the umbrella genre categories. We could classify the Digital Hardcore subgenre as belonging to more than one umbrella genre, but then the way songs are counted/filtered/color coded will need to change too. I think it makes sense to re-evaluate all those things, though.

**Comment by hvbish** [comment]
I wonder if we could use the attributes themselves to help us classify subgenres. As we've discovered, some genres separate out very nicely in parameter space.
